tests/test_create.py

```diff
@@ -1686,6 +1686,44 @@ def test_shortcut_absent_when_condarc_set():
         rmtree(prefix, ignore_errors=True)
         if isfile(shortcut_file):
             os.remove(shortcut_file)
+        # even if the $PREFIX/.condarc is gone,
+        # the context object still has that config in memory
+        reset_context()
+
+
+def test_menuinst_v2(monkeypatch: MonkeyPatch):
+    called = False
+    from menuinst import install
+
+    def mock_install(*args, **kwargs):
+        nonlocal called, install
+        called = True
+        return install(*args, **kwargs)
+
+    monkeypatch.setattr("menuinst.install", mock_install)
+    prefix = make_temp_prefix(str(uuid4())[:7])
+
+    Path(prefix).mkdir(parents=True, exist_ok=True)
+    Path(prefix).touch(".nonadmin")
+    out, err, _ = run_command(
+        Commands.CREATE,
+        prefix,
+        "jaimergp/label/menuinst-tests::package_1",
```

Comment:
This should be probably moved to a conda-owned channel. [Sources are in the menuinst repo](https://github.com/conda/menuinst/tree/main/tests/data/pkgs).